### PR TITLE
Adding the ability to send both Live and VOD analytics data separately.

### PR DESCRIPTION
**Pull request by thaveedu1**
Just adding unit tests to the pull request #90 ,

### CODE REVIEW

File: js/plugins/omniture.js [hunk 1]
```diff
@@ -647,10 +652,14 @@ var OoyalaPlayerDelegate = function()
...
     videoInfo.length = length;
-    //TODO: StreamType and update unit test
-    //The type of the video asset, one of the following: AssetType.ASSET_TYPE_LIVE,
-    //AssetType.ASSET_TYPE_LINEAR, AssetType.ASSET_TYPE_VOD
-    videoInfo.streamType = ADB.va.plugins.videoplayer.AssetType.ASSET_TYPE_VOD;
+    if (streamType === OO.Analytics.STREAM_TYPE.LIVE_STREAM) {
+      videoInfo.streamType = ADB.va.plugins.videoplayer.AssetType.ASSET_TYPE_LIVE;
+    } else if (streamType === OO.Analytics.STREAM_TYPE.VOD) {
+      videoInfo.streamType = ADB.va.plugins.videoplayer.AssetType.ASSET_TYPE_VOD;
+    }else{
```
Comment (aeng7) [inline js/plugins/omniture.js]: Minor issue - spacing. Only fix if there are other things to fix.

File: test/unit-tests/testOmniture.js [hunk 2]
```diff
@@ -264,6 +264,72 @@ describe('Analytics Framework Omniture Plugin Unit Tests', function()
     expect(videoInfo.playerName).toBe(playerName);
     expect(videoInfo.playhead).toBe(10);
   });
+it('Delegate can provide Video Info with VOD as the default streamType', function()
```
Comment (aeng7) [inline test/unit-tests/testOmniture.js]: Should be two spaces before 'it'

**Comment by aeng7** [comment]
@traja-ooyala looks good! Please fix the few style issues while we wait for another review (we require two reviews to merge).

File: js/plugins/omniture.js (hunk 1 above)
Comment (traja-ooyala) [reply js/plugins/omniture.js]: Done the suggested style changes

**Comment by pilievOoyala** [comment]
Looks good to me.

**Pull request merged**